monitor-openqa_job

```diff
@@ -14,31 +16,26 @@ host="${host:-"https://openqa.opensuse.org"}"
 openqa_groupid="${openqa_groupid:-"24"}"
 obs_component="${obs_component:-"package"}"
 
-# shellcheck source=/dev/null
 . "$(dirname "$0")"/_common
 
-# note: JOB_ID can return multiple job ids separated by newline which is not
-# properly handled here, instead implicitly only the first job is evaluated,
-# hence SC2086 is so far not attended. The proper approach would be to monitor
-# all jobs or just accept a single job and not ignore the style warning.
-
-[ -f job_post_response ] || (echo "Need job response status file 'job_post_response'" && exit 2)
-JOB_ID=$(filter_id job_post_response)
-echo "waiting for job ${JOB_ID} to finish"
-while sleep "${sleep_time}" ; do
-    # shellcheck disable=SC2086
-    [ "$($openqa_cli api --host "${host}" jobs/"${JOB_ID}" | jq -r '.job.state')" = 'done' ] && break
+[[ -f job_post_response ]] || (echo "Need job response status file 'job_post_response'" && exit 2)
+failures=0
+for job_id in $(job_ids job_post_response); do
+    echo "Waiting for job $job_id to finish"
+    while sleep "${sleep_time}" ; do
+        job_state=$($openqa_cli api --host "$host" jobs/"$job_id" | jq -r '.job.state')
+        [[ $job_state = 'done' ]] && break
+    done
+    response=$($openqa_cli api jobs/"$job_id")
+    result=$(echo "$response" | jq -r '.job.result')
+    echo "Result of job $job_id: $result"
+    obs_package_name=$1
+    if [[ $result != 'passed' ]] && [[ -n $obs_package_name ]] ; then
+        VERSION=$(echo "$response" | jq -r '.job.settings.VERSION')
+        osc api /comments/"${obs_component}"/"$obs_package_name" | grep id= | sed -n 's/.*id="\([^"]*\)">test failed.*/\1/p' | while read -r id; do osc api -X DELETE /comment/"$id"; done
+        osc api --data="test failed, see https://openqa.opensuse.org/tests/overview?version=$VERSION&groupid=${openqa_groupid}" -X POST "/comments/${obs_component}/$obs_package_name"
+        failures=$((failures + 1))
```

Comment:
So we would add a comment on OBS for each failing test but always referencing the overview page. I think we can do better :)

As you already count failures how about instead of adding up a number add all failed jobs to an array and if the array is not empty put a comment on OBS and exit with 1